lib/llm/providers/openai.rb

```diff
@@ -30,8 +30,18 @@ def complete(message, **params)
       params = DEFAULT_PARAMS.merge(params)
       body = {messages: messages.map(&:to_h)}.merge!(params)
       req = preflight(req, body)
-      res = request(@http, req)
-      Response::Completion.new(res.body, self).extend(response_parser)
+      if params[:stream]
```

Comment:
We could offload the complexity to another method
```ruby
stream!(req, body) if params[:stream]
```

Reply:
@0x1eef I'm still not sure about we go about this